common/app/views/fragments/immersiveMainMedia.scala.html

```diff
@@ -87,9 +87,18 @@
             <div class="gs-container">
                 @if(!isTheMinuteArticle) {
                     <div class="content__main-column">
-                    @fragments.meta.metaInline(page.item)
+                    @if(!isPaidContent){
+                      @fragments.meta.metaInline(page.item)
+                    } else {
+                      <div class="section-label--advertisement">
```

Comment:
I guess this should be `content__section-label content__section-label--advertisement` eventually

👮